type-definitions/dom-event.d.ts

```diff
@@ -2,29 +2,29 @@ import {Stream} from "most";
-export function mousemove(node: EventTarget, capture?: boolean): Stream<Event>;
-export function mouseover(node: EventTarget, capture?: boolean): Stream<Event>;
-export function mouseenter(node: EventTarget, capture?: boolean): Stream<Event>;
-export function mouseout(node: EventTarget, capture?: boolean): Stream<Event>;
-export function mouseleave(node: EventTarget, capture?: boolean): Stream<Event>;
+export function blur(node: EventTarget, capture?: boolean): Stream<FocusEvent>;
+export function focus(node: EventTarget, capture?: boolean): Stream<FocusEvent>;
+export function focusin(node: EventTarget, capture?: boolean): Stream<FocusEvent>;
+export function focusout(node: EventTarget, capture?: boolean): Stream<FocusEvent>;
+export function click(node: EventTarget, capture?: boolean): Stream<MouseEvent>;
+export function dblclick(node: EventTarget, capture?: boolean): Stream<MouseEvent>;
+export function mousedown(node: EventTarget, capture?: boolean): Stream<MouseEvent>;
+export function mouseup(node: EventTarget, capture?: boolean): Stream<MouseEvent>;
+export function mousemove(node: EventTarget, capture?: boolean): Stream<MouseEvent>;
+export function mouseover(node: EventTarget, capture?: boolean): Stream<MouseEvent>;
+export function mouseenter(node: EventTarget, capture?: boolean): Stream<MouseEvent>;
+export function mouseout(node: EventTarget, capture?: boolean): Stream<MouseEvent>;
+export function mouseleave(node: EventTarget, capture?: boolean): Stream<MouseEvent>;
 export function change(node: EventTarget, capture?: boolean): Stream<Event>;
-export function select(node: EventTarget, capture?: boolean): Stream<Event>;
+export function select(node: EventTarget, capture?: boolean): Stream<UIEvent>;
 export function submit(node: EventTarget, capture?: boolean): Stream<Event>;
-export function keydown(node: EventTarget, capture?: boolean): Stream<Event>;
-export function keypress(node: EventTarget, capture?: boolean): Stream<Event>;
-export function keyup(node: EventTarget, capture?: boolean): Stream<Event>;
+export function keydown(node: EventTarget, capture?: boolean): Stream<KeyboardEvent>;
+export function keypress(node: EventTarget, capture?: boolean): Stream<KeyboardEvent>;
+export function keyup(node: EventTarget, capture?: boolean): Stream<KeyboardEvent>;
 export function input(node: EventTarget, capture?: boolean): Stream<Event>;
-export function contextmenu(node: EventTarget, capture?: boolean): Stream<Event>;
-export function resize(node: EventTarget, capture?: boolean): Stream<Event>;
-export function scroll(node: EventTarget, capture?: boolean): Stream<Event>;
+export function contextmenu(node: EventTarget, capture?: boolean): Stream<PointerEvent>;
+export function resize(node: EventTarget, capture?: boolean): Stream<UIEvent>;
+export function scroll(node: EventTarget, capture?: boolean): Stream<UIEvent>;
 export function error(node: EventTarget, capture?: boolean): Stream<Event>;
```

Comment:
I wasn't quite sure where to look for these TS types, but google led me [heree](https://github.com/Microsoft/TypeScript/blob/master/src/lib/dom.generated.d.ts#L4209).  All of the occurrences of the `error` event I could find there seem to be of type `ErrorEvent`.  If that's right, then let's update it here, too.

Reply:
Skipped. Done.